### ISSUE TITLE
CHS merger updates for the Scientist page

### ISSUE THREAD

**Issue by mekline** [lookit/lookit-api#1180]
(see comment from MKS below with changes!)
# TL;DR

Scientists (https://lookit.mit.edu/scientists/)
  `web/templates/web/scientists/core-team.html`, `web/templates/web/scientists/working-groups.html`, `web/templates/web/scientists/parent-researcher-collaborative.html`
  _"Meet the **Lookit** team"_ (Are we the Lookit team or CHS team?)
  _"Core **Lookit** Team"_ 
  _"Although the Core Team at MIT provides a foundation for **Lookit** to function, a lot of great additional work is done by the community of researchers using **Lookit**."_
    _"supporting new researchers as they learn how to use **Lookit**."_

Scientists (https://lookit.mit.edu/scientists/)
    `web/templates/web/scientists/parent-researcher-collaborative.html`
    _"If you are a parent who has participated with your child in a study on this website, or a university-based researcher who has posted a study on **Lookit** or **childrenhelpingscience.com**, then we consider you a member of the Parent Researcher Collaborative!"_

**Comment by mekline** [comment]
Updates: 

"Meet the Children Helping Science Team!"

Core Lookit team --> "Core Team"

- Becky is a software engineer for **Children Helping Science**...
- CJ is **Children Helping Science**'s lead software infrastructure engineer...
- Melissa is **the Executive Director of Children Helping Science**....how to make **Children Helping Science** better.
- (I will ask Mark for a new bio as we are not currently running WG groups under either name!)

Parent Researcher Collaborative:

...a university-based researcher who has posted **a study on Children Helping Science (or its previous incarnation as Lookit)**, then...

Affiliated Universities 

As of March 2023, there are over 950 researchers in our community, and over 90 universities affiliated with **Lookit/Children Helping Science**, including: 


(note the nomenclature difference here since the Lookit name was used for the initial agreement)

And please *drop* the Working Groups section - probably just comment this out as we will hopefully repopulate it in the next 3-6 months, but for now it doesn't make sense to talk about this group as members of "CHS working groups"
